### PR TITLE
Fix reference to stable/* for influxdata/*

### LINKED ISSUE #13: Fix Usage of "stable/*" on Individual Chart READMEs

### PR DESCRIPTION
**Pull request by nsteinmetz**
Closes #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
 Once Helm is set up properly, add the repo as follows:
 
 ```console
-helm repo add influxdata https://helm.influxdata.com
+$ helm repo add influxdata https://helm.influxdata.com/
```
Comment (naseemkullah) [inline README.md]: As per https://github.com/updownpress/markdown-lint/blob/master/rules/014-commands-show-output.md, dollar signs should not precede shell commands if no output is shown. Could we remove them from all such cases?

Comment (nsteinmetz) [reply README.md]: Ah ok, it was to be consistent with all other code samples ; I'll fix all of them then now I know the rule :)

Comment (nsteinmetz) [reply README.md]: Fixed !

**Comment by nsteinmetz** [comment]
I don't get the point about the conflict:

```
<<<<<<< fix_stable_reference
version: 4.2.9
=======
version: 4.2.8
>>>>>>> master
```

**Comment by nsteinmetz** [comment]
Ok, seems I missed a rebase/master somehow

**Comment by naseemkullah** [comment]
cc @rawkode @gitirabassi

**Comment by nsteinmetz** [comment]
@naseemkullah sorry, I just rebase on most recent master so it removed your approval...

**Comment by rawkode** [comment]
Thanks @nsteinmetz

**Pull request merged**